src/OSPSuite.Presentation/Presenters/ParameterIdentifications/ParameterIdentificationSingleRunAnalysisPresenter.cs

```diff
@@ -40,6 +40,7 @@ public ParameterIdentificationRunResult SelectedRunResults
          {
             UpdateTemplateFromChart();
             updateSelectedRunResults(value);
+            UpdateChartFromTemplate();
```

Comment:
This was missing. We were saving the template but never loading from template. It was working implicitly before probably...somehow